src/js/api.js

```diff
@@ -1,26 +1,36 @@
 import fetch from 'isomorphic-fetch'
 
-function sendFetch(suffix, options = {method: 'GET'}){
-  var url = makeUrl(parseLocation(), suffix);
+let accessToken
 
+function sendFetch(suffix, options = {method: 'GET'}){
+  var location = options.noParse ? {} : parseLocation();
+  var url = makeUrl(location, suffix);
   if(typeof options.body === 'object') options.body = JSON.stringify(options.body)
+  var headers = {}
 
+  if(accessToken) headers.Authorization = 'Bearer ' + accessToken
+  console.log('requesting with access token: ', accessToken)
   var fetchOptions = {
-    method: options.method,
+    method: options.method || 'GET',
     body: options.body,
-    headers: {
-      'CFPB-HMDA-Institutions': '0,1,2,3',
-      'CFPB-HMDA-Username': 'fakeuser',
-      'Content-Type': 'application/json'
-    }
+    headers: headers
   }
 
+  console.log('fetching from', url, 'with options', fetchOptions)
   return fetch(url, fetchOptions)
-    .then(response => response.json())
+    .then(response => {console.log('respose from fetch', response);return response.json()})
+}
+
+export function setAccessToken(token) {
+  accessToken = token
+}
+
+export function getAccessToken() {
+  return accessToken || ''
 }
 
 export function makeUrl(obj, suffix){
-  var url = location.protocol + '//' + location.host + '/api'
+  var url = process.env.HMDA_API || location.protocol + '//' + location.host + '/hmda'
```

Comment:
Probably never. That was just put in place as a sane-ish default (which might actually be worse because it will be less obvious what's happening in case of failure)